src/main/java/gregtech/common/blocks/BlockFrame.java

```diff
@@ -113,43 +100,15 @@ public int getHarvestLevel(@Nonnull IBlockState state) {
         return 1;
     }
 
-    @Override
-    protected BlockStateContainer createStateContainer() {
-        return new BlockStateContainer(this, variantProperty);
-    }
-
     @Override
     @Nonnull
     @SuppressWarnings("deprecation")
     public net.minecraft.block.material.Material getMaterial(IBlockState state) {
-        Material material = state.getValue(variantProperty);
+        Material material = getGtMaterial(state);
         if (ModHandler.isMaterialWood(material)) {
             return net.minecraft.block.material.Material.WOOD;
         }
-        return super.getMaterial(state);
-    }
-
-    @Override
-    public void getSubBlocks(@Nonnull CreativeTabs tab, @Nonnull NonNullList<ItemStack> list) {
-        blockState.getValidStates().stream()
-                .filter(blockState -> blockState.getValue(variantProperty) != Materials.NULL)
-                .forEach(blockState -> list.add(getItem(blockState)));
-    }
-
-    public static ItemStack getItem(IBlockState blockState) {
-        return GTUtility.toItem(blockState);
-    }
-
-    public ItemStack getItem(Material material) {
-        return getItem(getDefaultState().withProperty(variantProperty, material));
-    }
-
-    public IBlockState getBlock(Material material) {
-        return getDefaultState().withProperty(variantProperty, material);
-    }
-
-    public Material getGtMaterial(int meta) {
-        return variantProperty.getAllowedValues().get(meta);
+        return net.minecraft.block.material.Material.IRON;
```

Comment:
Hm, why this change? I know that super should always return Material.IRON because that is set in the constructor, but I would rather just continue calling super in case we change materials on frames, or just some frames, some day